### ISSUE TITLE
Compatibility on Android

### ISSUE THREAD

**Issue by taylortom** [cgkineo/jquery.a11y#6]
Tried testing this on Android, and is almost unusable (due to the poor TalkBack app). From our experience internally, users with accessibility requirements opt not to use Android for this reason.

Just wanted to post this to see if anyone has any differing opinions on the matter, as Android compatibility is relatively low on the agenda as it stands.

**Comment by olivermartinfoster** [comment]
Yep, I need to know if it's worth investing the time in getting this working. Please give a yay or a ney and any other requirements you may have.

**Comment by moloko** [comment]
I would say that it's not - for this release anyway. Can always be added in the future.

Matt Leathes
